WireDomain/Sources/WireDomain/Repositories/UpdateEvents/Protocols/UpdateEventsLocalStoreProtocol.swift

```diff
@@ -63,17 +63,17 @@ public protocol UpdateEventsLocalStoreProtocol {
 
     /// Fetches stored event envelopes.
     /// - parameter limit: A fetch limit.
-    /// - returns: A list of event envelopes.
+    /// - returns: A list of decoded event envelopes and their related object IDs.
 
     func fetchStoredEventEnvelopes(
         limit: UInt
-    ) async throws -> [UpdateEventEnvelope]
+    ) async throws -> [(UpdateEventEnvelope, objectID: NSManagedObjectID)]
```

Comment:
suggestion: make tuple element explicit
```suggestion
    ) async throws -> [(envelope: UpdateEventEnvelope, objectID: NSManagedObjectID)]
```

Reply:
done https://github.com/wireapp/wire-ios/pull/3173/commits/720fdb56505f79b80a56cb3278b3bb584e1777ce